crawl-ref/source/dat/des/branches/temple.des

```diff
@@ -2985,6 +2985,85 @@ cc.....cccc+ccccccccccc...B...cc.....cccc+ccc
                                   ccccc              
 ENDMAP
 
+NAME:    mainiacjoe_temple_starburst
+PLACE:   Temple
+ORIENT:  encompass
+WEIGHT:  10
+SUBST:   c : c
```

Comment:
This SUBST doesn't appear to do anything. I am not opposed to just sitting around doing nothing _in principle_, mind you, but this is the wrong place for a SUBST to do that.